test/e2e/tinkerbell_test.go

```diff
@@ -1516,6 +1596,71 @@ func TestTinkerbellKubernetes127UbuntuSimpleFlow(t *testing.T) {
 	runTinkerbellSimpleFlow(test)
 }
 
+func TestTinkerbellKubernetes123Ubuntu2204SimpleFlow(t *testing.T) {
+	provider := framework.NewTinkerbell(t)
+	test := framework.NewClusterE2ETest(
+		t,
+		provider,
+		framework.WithClusterFiller(api.WithKubernetesVersion(v1alpha1.Kube123)),
+		framework.WithControlPlaneHardware(1),
+		framework.WithWorkerHardware(1),
+	)
+	test.WithClusterConfig(provider.WithKubeVersionAndOS(v1alpha1.Kube123, framework.Ubuntu2204, nil))
```

Comment:
Ah I see how passing the release is annoying for tests that just want to use main.
Could we create a ticket to address this? Probably creating some kind of helper that only takes kube version and OS for this kind of tests.